src/main/scala/io/iohk/ethereum/mining/BlockGenerator.scala

```diff
@@ -40,7 +40,8 @@ class BlockGenerator(blockchainStorages: BlockchainStorages, blockchainConfig: B
           ommersHash = ByteString(kec256(ommers.toBytes: Array[Byte])),
           beneficiary = beneficiary.bytes,
           stateRoot = ByteString.empty,
-          transactionsRoot = buildMpt(transactions, SignedTransaction.byteArraySerializable),
+          //not here
```

Comment:
Ok, maybe we can improve this comment because it seems a message you forgot to remove. What about something like _unknown at this point_ ?